example/driver/simpledriver.go

```diff
@@ -74,11 +78,55 @@ func (s *SimpleDriver) Initialize(lc logger.LoggingClient, asyncCh chan<- *dsMod
 	s.lc = lc
 	s.asyncCh = asyncCh
 	s.deviceCh = deviceCh
+	s.serviceConfig = &config.ServiceConfig{}
+
+	ds := service.RunningService()
+
+	if err := ds.LoadCustomConfig(s.serviceConfig, "SimpleCustom"); err != nil {
+		return fmt.Errorf("unable to load 'SimpleCustom' custom configuration: %s", err.Error())
+	}
+
+	lc.Infof("Custom config is: %v", s.serviceConfig.SimpleCustom)
+
+	if err := s.serviceConfig.SimpleCustom.Validate(); err != nil {
+		return fmt.Errorf("'SimpleCustom' custom configuration validation failed: %s", err.Error())
+	}
+
+	if err := ds.ListenForCustomConfigChanges(
+		&s.serviceConfig.SimpleCustom.Writable,
+		"SimpleCustom/Writable", s.ProcessCustomConfigChanges); err != nil {
+		return fmt.Errorf("unable to listen for changes for 'SimpleCustom.Writable' custom configuration: %s", err.Error())
+	}
+
 	return nil
 }
 
+// ProcessCustomConfigChanges ...
+func (s *SimpleDriver) ProcessCustomConfigChanges(rawWritableConfig interface{}) {
+	updated, ok := rawWritableConfig.(*config.SimpleWritable)
+	if !ok {
+		s.lc.Error("unable to process custom config updates: Can not cast raw config to type 'SimpleWritable'")
+		return
+	}
+
+	s.lc.Info("Received configuration updates for 'SimpleCustom.Writable' section")
+
+	previous := s.serviceConfig.SimpleCustom.Writable
+	s.serviceConfig.SimpleCustom.Writable = *updated
+
+	if reflect.DeepEqual(previous, *updated) {
+		s.lc.Info("No changes detected")
+		return
+	}
+
+	if previous.DiscoverSleepDurationSecs != updated.DiscoverSleepDurationSecs {
+		s.lc.Infof("DiscoverSleepDurationSecs changed to: %d", updated.DiscoverSleepDurationSecs)
+	}
+}
+
 // HandleReadCommands triggers a protocol Read operation for the specified device.
-func (s *SimpleDriver) HandleReadCommands(deviceName string, protocols map[string]contract.ProtocolProperties, reqs []dsModels.CommandRequest) (res []*dsModels.CommandValue, err error) {
+func (s *SimpleDriver) HandleReadCommands(
+	deviceName string, protocols map[string]contract.ProtocolProperties, reqs []dsModels.CommandRequest) (res []*dsModels.CommandValue, err error) {
```

Comment:
Why did you change this? If it's a result of your IDE formatting, please revert and address formatting in a separate commit/PR.

Reply:
Not sure. Reverted.